astropy/units/core.py

```diff
@@ -446,6 +499,73 @@ def add_enabled_equivalencies(equivalencies):
     return context
 
 
+def set_enabled_aliases(aliases):
+    """
+    Set aliases for units, e.g., to handle alternate spellings.
```

Comment:
Hm, I guess same comment as above.

Reply:
I ended up thinking the sentence was mostly useful for the functions, much less so in the method (which the user essentially never sees...)